hazenlib/utils.py

```diff
@@ -460,7 +455,9 @@ def determine_orientation(dcm_list):
             return "axial", z
         else:
             logger.warning("Unable to determine orientation based on DICOM metadata")
-            logger.info("x %s\ny %s\nz %s", set(x), set(y), set(z))
```

Comment:
I think the original implementation here is more concise so I would revert back to that. Is there a reason for splitting up the logging messages?